.changeset/tall-garlics-change.md

```diff
@@ -0,0 +1,11 @@
+---
+"@razorpay/blade": major
+---
+
+feat(Link): add size small
```

Comment:
nit: 
```suggestion
feat(Link): add `size` prop and support `small` size
```

Reply:
Done in https://github.com/razorpay/blade/pull/761/commits/e35ad717133247c3e00611f6fb5a3623efdb3ed4